static/app/views/replays/list/replaysErroneousDeadRageCards.tsx

```diff
@@ -191,37 +165,39 @@ function CardTable({
         sort={undefined}
         visibleColumns={visibleColumns}
         saveLocation
-        gridRows={'auto ' + gridRows}
+        gridRows={`auto ${repeat(' 1fr', rows)}`}
         showDropdownFilters={false}
       />
-      <StyledButton
-        size="sm"
-        onClick={() => {
-          const newQuery =
-            emptyLocation.query.query === searchQuery.query
-              ? emptySearchQuery
-              : searchQuery;
-          browserHistory.push({
-            pathname: emptyLocation.pathname,
-            query: newQuery,
-          });
-        }}
-        icon={
-          emptyLocation.query.query === searchQuery.query ? (
-            <IconClose size="xs" />
-          ) : (
-            <IconSearch size="xs" />
-          )
-        }
-      >
-        {emptyLocation.query.query === searchQuery.query
-          ? t('Clear filter')
-          : buttonLabel}
-      </StyledButton>
+      {children}
     </Fragment>
   );
 }
 
+function SearchButton({eventView, label}: {eventView: EventView; label: ReactNode}) {
+  const location = useLocation();
+  const isActive = location.query.query === eventView.query;
+
+  return (
+    <StyledButton
+      size="sm"
+      onClick={() => {
+        browserHistory.push({
+          pathname: location.pathname,
+          query: {
+            ...location.query,
+            cursor: undefined,
+            query: isActive ? '' : eventView.query,
+            sort: isActive ? '' : eventView.sorts[0].field,
+          },
+        });
+      }}
```

Comment:
m: This should probably be a link (I don't remember the component for a link that looks like a button) instead?

Reply:
👍 LinkButton